src/components/MetadataForm/__stubs__/schema.ts

```diff
@@ -0,0 +1,21 @@
+export const schema = {
```

Comment:
Kan dit niet getyped worden?

Reply:
I wasn't able to figure out how in the case where custom keywords are thrown in the mix

Reply:
I found a way with a custom type